### PR TITLE
Add release notes for 14.9

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Client Release Notes 14.9:
+
+## Updated Resources
+
+### group
+
+On Windows hosts the group resource now supports setting the comment field via a new `comment` property.
```
Comment (mjingle) [inline RELEASE_NOTES.md]:
```suggestion
On Windows hosts, the group resource now supports setting the comment field via a new `comment` property.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+
+### homebrew_cask
+
+Two issues that caused homebrew_cask to converge on each Chef run have been resolved. Thanks [@jeroenj](https://github.com/jeroenj) for this fix. Additionally the resource will no longer fail if the `cask_name` property is specified.
```
Comment (mjingle) [inline RELEASE_NOTES.md]:
```suggestion
Two issues, which caused homebrew_cask to converge on each Chef run, have been resolved. Thanks [@jeroenj](https://github.com/jeroenj) for this fix. Additionally, the resource will no longer fail if the `cask_name` property is specified.
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,65 @@
...
+
+### homebrew_tap
+
+The homebrew_tap resource no longer fails if the `tap_name` property is specified.
+
+### openssl_x509_request
+
+The openssl_x509_request resource now property writes out the CSR file if the `path` property is specified. Thank you [@cpjones](https://github.com/cpjones) for reporting this issue.
```
Comment (mjingle) [inline RELEASE_NOTES.md]:
```suggestion
The openssl_x509_request resource now properly writes out the CSR file if the `path` property is specified. Thank you [@cpjones](https://github.com/cpjones) for reporting this issue.
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,65 @@
...
+
+### powershell_package_source
+
+powershell_package_source now suppresses warnings which prevented properly loading the resource state, and resolves idempotency issues when both the `name` and `source_name` properties were specified. Thanks [@Happycoil](https://github.com/Happycoil) for this fix.
```
Comment (mjingle) [inline RELEASE_NOTES.md]:
```suggestion
powershell_package_source now suppresses warnings, which prevented properly loading the resource state, and resolves idempotency issues when both the `name` and `source_name` properties were specified. Thanks [@Happycoil](https://github.com/Happycoil) for this fix.
```

Comment (mjingle) [reply RELEASE_NOTES.md]: Is "idempotency" normally one word? Asking since my curiosity got the better of me.

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,65 @@
...
+
+The sysctl resource now allows slashes in the key or block name. This allows keys such as `net/ipv4/conf/ens256.401/rp_filter` to be used with this resource.
+
+### windows_ad_join
+
+Errors joining the domain are now properly suppressed from the console and logs if the `sensitive` property is set to true. Thanks [@Happycoil](https://github.com/Happycoil) for this improvement.
+
+### windows_certificate
+
+The delete action now longer fails if a certificate does not exist on the system. Additionally certificates with special characters in their passwords will no longer fail. Thank you for reporting this [@chadmccune](https://github.com/chadmccune)
```
Comment (mjingle) [inline RELEASE_NOTES.md]:
```suggestion
The delete action now longer fails if a certificate does not exist on the system. Additionally, certificates with special characters in their passwords will no longer fail. Thank you for reporting this [@chadmccune](https://github.com/chadmccune).
```

File: RELEASE_NOTES.md (hunk 4 above)
Comment (tas50) [reply RELEASE_NOTES.md]: Yes? It's a frequently used word in config management just like that, but not anywhere else. Pretty sure Mark Burges is to blame for all of this.

**Pull request merged**